### PR TITLE
Check libdevice existence in HeavyDB server.

### PR DESCRIPTION
**Pull request by pearu**
As in the title.

Requires https://github.com/heavyai/heavydb-internal/pull/6914

### CODE REVIEW

File: rbc/heavydb/remoteheavydb.py [hunk 1]
```diff
@@ -1066,8 +1067,16 @@ def retrieve_targets(self):
                 target_info.add_library('stdio')
                 target_info.add_library('stdlib')
                 target_info.add_library('heavydb')
-            elif target_info.is_gpu and self.version >= (5, 5):
-                target_info.add_library('libdevice')
+            elif target_info.is_gpu:
+                print(f'{target_info.has_libdevice=}')
```
Comment (guilhermeleobas) [inline rbc/heavydb/remoteheavydb.py:1071]: remove print statement

File: rbc/tests/heavydb/test_math.py [hunk 2]
```diff
@@ -129,6 +129,9 @@ def test_math_function(heavydb, device, nb_version, fn_name, signature):
     if not heavydb.has_cuda and device == 'gpu':
         pytest.skip('test requires CUDA-enabled heavydb server')
 
+    if not heavydb.has_cuda_libdevice and device == 'gpu':
+        pytest.skip('Test requires CUDA-enabled heavydb server with libdevice')
+
```
Comment (guilhermeleobas) [inline rbc/tests/heavydb/test_math.py:134]: Update `rbc/tests/test_externals_libdevice.py` as well to skip the tests if libdevice is not available.

File: rbc/heavydb/remoteheavydb.py (hunk 1 above)
Comment (pearu) [reply rbc/heavydb/remoteheavydb.py:1071]: Fixed.

File: rbc/tests/heavydb/test_math.py (hunk 2 above)
Comment (pearu) [reply rbc/tests/heavydb/test_math.py:134]: Fixed.

**Pull request merged**